### ISSUE TITLE
SA1633 should allow multi-line comment file headers

### ISSUE THREAD

**Issue by Noryoko** [DotNetAnalyzers/StyleCopAnalyzers#1432]
The following examples produce no errors in classic, but are flagged by `FileHeaderAnalyzers`.

```csharp
/* <copyright file="Test.cs" company="Test">
 Copyright (c) Test. All rights reserved.
 </copyright> */
```

```csharp
/*
 <copyright file="Test.cs" company="Test">
 Copyright (c) Test. All rights reserved.
 </copyright>
*/
```

**Comment by vweijsters** [comment]
Grabbing this
